webapp/src/positionPicker.tsx

```diff
@@ -0,0 +1,241 @@
+import * as React from 'react';
```

Comment:
does it have to be in webapp/src/ ? spriteEditor has a folder in pxtlib I think. Preferably this should go in a folder with other snippet builder specific stuff